modules/stratos/chart.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+. ../../include/common.sh
+. .envrc
+
+set -Eexuo pipefail
+debug_mode
+
+if [ -z "$STRATOS_CHART" ]; then
+    warn "No chart url given - using latest public release from GH"
+        STRATOS_CHART=$(curl -s https://api.github.com/repos/cloudfoundry/stratos/releases/latest | grep "browser_download_url.*zip" | cut -d : -f 2,3 | tr -d \" | tr -d " ")
```

Comment:
really nice! Probably we should move the gh fetch to its own function at some point

Reply:
(or even the whole chart fetch/uncompress)

Reply:
well that's entirely lifted from scf-chart :D . It works though.